lib/Test.js

```diff
@@ -256,6 +257,10 @@ define([
 				})
 				.then(function () {
 					self.hasPassed = true;
+					if (self._usesRemote && !self.isAsync) {
+						console.warn("This test uses the remote but isn't marked as async! Intern won't wait for your test actions to complete.");
```

Comment:
Messages should go through `reporterManager` so the user can handle and filter them, rather than sending messages directly to console.